magma/spec/retrieve_spec.rb

```diff
@@ -333,7 +333,7 @@ def retrieve(post, user_type=:viewer)
       )
 
       header, *table = CSV.parse(last_response.body, col_sep: "\t")
-      expect(table).to eq([ [ "The Twelve Labors of Hercules", labors.map(&:identifier).join(', ') ] ])
+      expect(table).to match_array([ [ "The Twelve Labors of Hercules", labors.map(&:identifier).join(', ') ] ])
```

Comment:
I see this spec failing intermittently because of the table order -- so changing the spec to use `match_array` instead of `eq` -- we don't expect the TSV data to come back in a specific order, do we?